root/about/sponsors.tx

```diff
@@ -319,7 +319,23 @@ community.
                 </td>
             </tr>
             <tr>
-                <td><a href="https://kritika.io"><img src="/static/images/sponsors/kritika.svg" class="image-responsive"></a></td>
+                <td><a href="https://deriv.com/careers/"><img src="/static/images/sponsors/deriv-horizontal-web-dark-100.jpeg" width="170" align="left"></a></td>
+                <td>
+                    <p>
+                        <a href="https://deriv.com">Deriv</a> is one of the world’s largest online brokers.
+                        <a href="https://deriv.com">Deriv</a> offers CFDs and other derivatives on forex,
+                        indices, cryptocurrencies, commodities, and synthetics to millions of registered users across the globe.
+                    </p>
+                    <p>
+                        More than that, <a href="https://deriv.com/careers/">Deriv</a> heavily relies on CPAN (and bring back a lot to it in return!)
```

Comment:
This link goes to a different URL, but has the same text as the others. Is this intentional?

Reply:
Good point I will change this